### PR TITLE
Pyfolio risk analysis

### PR DESCRIPTION
**Pull request by alen12345**

### CODE REVIEW

File: recipes/pyfolio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (CJ-Wright) [inline recipes/pyfolio/meta.yaml]: Could you please remove these comments?

File: recipes/pyfolio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  # noarch: python
```
Comment (CJ-Wright) [inline recipes/pyfolio/meta.yaml]: Could this be a noarch package?

File: recipes/pyfolio/meta.yaml (hunk 1 above)
Comment (alen12345) [reply recipes/pyfolio/meta.yaml]: Hi @CJ-Wright Do you want me to remove all the comments in the file or only specific ones?

File: recipes/pyfolio/meta.yaml (hunk 2 above)
Comment (alen12345) [reply recipes/pyfolio/meta.yaml]: Yes, it definitely is, I will change it in the next commit!

**Comment by alen12345** [comment]
Need to build other dependencies such as empyrical before proceeding with pyfolio.

**Comment by CJ-Wright** [comment]
Ok, but you don't have to close the PR, you can just have this one wait for those.

**Comment by alen12345** [comment]
I cannot find it in the feedstock in order to update the package with the license file and to remove the comments. Why?

**Comment by CJ-Wright** [comment]
It seems that the file was deleted before it could be moved to a feedstock. https://github.com/conda-forge/staged-recipes/commit/221c6879fcdb1bbb6293ccf0dd8b3f750e044de6

I'm not certain how that happened, did you push to the staged-recipes master?

**Comment by CJ-Wright** [comment]
Please submit a new PR in the meantime.

**Comment by alen12345** [comment]
I don't know what happened too; do you want a new PR from my forked master branch or my new pyfolio dedicated branch?

**Comment by alen12345** [comment]
It should be, let us try.

**Comment by CJ-Wright** [comment]
Please make certain to not push directly to staged-recipes

**Pull request merged**